src/schneider_lab_to_nwb/schneider_2024/schneider_2024_metadata.yaml

```diff
@@ -1,14 +1,55 @@
   institution: Institution where the lab is located
   lab: Schneider
   experimenter:
-    - Last, First Middle
-    - Last, First Middle
+    - Zempolich, Grant W.
+    - Schneider, David M.
 Subject:
-  species: Rattus norvegicus
-  age: TBD  # in ISO 8601, such as "P1W2D"
-  sex: TBD  # One of M, F, U, or O
+  species: Mus musculus
+  age: P12W/  # in ISO 8601, such as "P1W2D"
+  sex: U  # One of M, F, U, or O
+  description: 12-week-old C57BL/6 or VGATChR2-EYFP mice (see Aims approaches below for details) will be used for all behavioral, electrophysiology, and optogenetic experiments. In the VGAT-ChR2-EYFP mouse line, channelrhodopsin (ChR2) is coupled to the vesicular GABA transporter, inducing expression in GABAergic inhibitory neurons ubiquitously across cortex and allowing for real time optogenetic inhibition of brain regions of interest.
+  strain: C57BL/6
+
+Ecephys:
+  Device:
+    - name: DeviceEcephys
+      description: Masmanidis Lab dense 128-channel recording probes (https://masmanidislab.neurobio.ucla.edu/technology.html).
+      manufacturer: Masmanidis Lab
+  ElectrodeGroup:
+    - name: ElectrodeGroup
+      description: ElectrodeGroup for all channels in the recording probe.
+      location: Auditory Cortex
+      device: DeviceEcephys
+  ElectricalSeries:
+    - name: ElectricalSeries
+      description: Recording of AC neural responses in mice performing this behavioral task will utilize dense 128-channel recording probes (Masmanidis Lab). These recording probes span a depth ~1mm, allowing for sampling of all layers of cortex. Electrophysiology data will be recorded using OpenEphys Acquisition Board v2.4 and associated OpenEphys GUI software.
+  UnitProperties:
+    - name: units
+      description: Neural spikes will be sorted offline using Kilosort 2.5 and Phy2 software and manually curated to ensure precise spike time acquisition.
```

Comment:
```suggestion
      description: Neural spikes was be sorted offline using Kilosort 2.5 and Phy2 software and manually curated to ensure precise spike time acquisition.
```

?

Reply:
I'm copying verbatim from a document sent by the Schneider lab...

We will want to change the tense before final conversion, but I think I'd rather do it all in one go.

Reply:
See #5

Reply:
OK.